CMakeLists.txt

```diff
@@ -12,13 +12,10 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 add_compile_definitions(PLATFORM_CPU_SISD)
-set(CMAKE_CXX_FLAGS "-O3 -Wall -march=native -mtune=native")
-
-IF (WIN32)
-    set(patch_file ${CMAKE_CURRENT_BINARY_DIR}/_deps/lbm-src/src/core/singleton.h)
-    set(lbm_patch sed -i -f ${CMAKE_CURRENT_SOURCE_DIR}/script.sed ${patch_file})
+IF (!WIN32)
+    set(CMAKE_CXX_FLAGS "-O3 -Wall -march=native -mtune=native")
```

Comment:
Without this line, the Eigen CMake module will complain because it fails to compile its compiler detection code. Must be some CMake weirdness that the flags bleed down into the dependencies. :)

The issue seems to be resolved on `master` of Eigen, but they do not seem to provide fixed tags anymore. :(
It could be considered to switch to a commit hash as a fixed reference point for a newer version. The dropped support of C++03 shouldn't matter too much since this repository uses C++17 anyway.

However, it works as-is - just pretty confusing at first if it doesn't work anymore just because this line is missing.